schemas/edges/GO_ontology/GO_vertices.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "GO_vertices",
+  "type": "object",
+  "description": "A entry for vertices in the Gene Ontology (GO) hierarchy",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "The unique id of the current term.",
+      "examples": ["GO:0022609", "GO:0044848"]
+    },
+    "name": {
+      "type": "string",
+      "description": "The term name. Any term may have only one name defined.",
+      "examples": ["mitochondrial genome maintenance", "reproduction"]
+    },
+    "alt_id": {
+      "type": "string",
+      "description": "Defines an alternate id for this term. A term may have any number of alternate ids.",
+      "examples": ["GO:0022609", "GO:0044848"]
+    },
+    "def": {
+      "type": "string",
+      "description": "The definition of the current term. ",
+      "examples": ["'The directed movement of a ribosomal subunit from the nucleus into the cytoplasm.' [GOC:ai]"]
+    },
+    "comment": {
+      "type": "string",
+      "description": "A comment for this term.",
+      "examples": ["This term was made obsolete because it refers to a class of gene products and a biological process rather than a molecular function."]
+    },
+    "subset": {
+      "type": "string",
+      "description": "This tag indicates a term subset to which this term belongs.",
+      "examples": ["goslim_pir", "goslim_plant"]
+    },
+    "synonym": {
```

Comment:
Does this need to be parsed further? also looks like a list

Reply:
synonyms contain typedefs after the name. I think we can leave it as is unless it would be useful to query whether a synonym is exact or related.